src/config.ts

```diff
@@ -24,6 +28,7 @@ export interface XdcZeroConfig {
   subnetZeroContractAddress?: string;
   parentChainZeroContractAddress?: string;
   walletPk?: string;
+  subnetWalletPk?: string;
```

Comment:
what's the difference between `subnetWalletPk` and `accountPk`?

Reply:
this part is for Zero specific wallet
we need diff wallets for Zero and CSC because they will clash if TXs are push at the same time.

Reply:
not sure that answers ur question